.github/workflows/build.yaml

```
@@ -1,6 +1,5 @@
 name: build
 on:
```

Maybe we wanna run the workflow when a `merge` happens? I'm not sure if the `pull_request` will run it at `merge` time, if it doesn't maybe we wanna keep something like this?

```suggestion
on:
  push:
    branches:
      - master  
```